src/components/Player/Player.jsx

```diff
@@ -7,11 +7,13 @@ import Long from 'long';
 import {
   getPlayer,
   getPlayerWinLoss,
+  getProPlayers,
```

Comment:
I don't think we need to fetch the entire list of pro players. The player response should have a "name" field which is the pro player name if available. If this is set, the player is a pro.

Reply:
I think this is what "officialPlayerName" is here

Reply:
import not needed?

Reply:
import still not needed?